lua/bmessages.lua

```diff
@@ -10,6 +10,7 @@ local function with_defaults(options)
 	local defaults = {
 		timer_interval = 1000,
 		split_type = "vsplit",
+		split_direction = "topleft",
```

Comment:
This does change the current default behavior lets make it `nil` by default. Users can opt into this feature by setting it.